crates/medea-reactive/src/field/mod.rs

```diff
@@ -27,6 +29,13 @@ type DefaultSubscribers<D> = RefCell<Vec<UniversalSubscriber<D>>>;
 /// ([`ObservableField::when`] and [`ObservableField::when_eq`]).
 pub type Observable<D> = ObservableField<D, DefaultSubscribers<D>>;
 
+/// [`ObservableField`] that allows to subscribe to all changes
+/// ([`ObservableField::subscribe`]) and to concrete changes
+/// ([`ObservableField::when`] and [`ObservableField::when_eq`]).
+///
+/// Can recognise when all updates was processed by subscribers.
+pub type ProgressableField<D> = ObservableField<D, progressable::SubStore<D>>;
```

Comment:
I guess it's better to name just `Progressable` to play well along with `Observable`.